src/utils/getFilterredTodos.ts

```diff
@@ -0,0 +1,19 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export function getFilteredTodos(todos: Todo[], filter: Filter) {
+  const filteredTodos = [...todos];
```

Comment:
The line creating `filteredTodos` as a copy of `todos` is unnecessary. The `filter` method does not mutate the original array, so you can directly filter `todos`.